Makefile

```diff
@@ -40,3 +45,9 @@ rpm:
 		printf "\nThe tendrl-api RPMs are located at:\n\n"; \
 		printf "   $(DEPLOY)/latest\n\n\n\n"; \
 	fi
+
+update-release:
+	sed -i tendrl-api.spec \
+	  -e "/^Release:/cRelease: $(shell date +"%m_%d_%Y_%H_%M_%S")"
```

Comment:
Please see my comment under the related issue related to the version format https://github.com/Tendrl/api/issues/209#issuecomment-314371093.